### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by TinasheMTapera**
Waiting on reviews

### CODE REVIEW

File: docs/LabHome/Onboarding/analyst_onboarding.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+
+Other resources:
+
+- [Swirl](https://swirlstats.com/students.html), an interactive prompt-based learning package
+- [learnR4free](https://www.learnr4free.com/index.html) and [this blog](https://livefreeordichotomize.com/2020/07/02/so-you-want-to-learn-r/) collate a number of external resources
+
+### Git & Github
+
+Version control with Git and Github is essential for managing reproducible science. The Informatics team handles all of their projects using git, so fluency with basic git commands and familiarity with a Github-based workflow are daily requirements.
+
```
Comment (mattcieslak) [inline docs/LabHome/Onboarding/analyst_onboarding.md:101]: Maybe add something about pull requests?

File: docs/LabHome/Onboarding/analyst_onboarding.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+### BIDS
+
+Brain Imaging Data Structure is a data storage standard for neuroimaging. It describes how best to format and store neuroimaging data (and why it's important to do so). PennLINC expects all of its data, both internal and publicly shared, to be in BIDS format — much of the work of the Informatics team is making sure this goal is being met.
+
+> Q: What are the benefits of using BIDS?
+
+> Q: What modalities are currently covered in BIDS? What happens if the data you have isn't covered in BIDS?
+
+We recommend learning about BIDS straight from the source: the [official BIDS specification](https://bids-specification.readthedocs.io/en/stable/) (< 1 hour).
```
Comment (mattcieslak) [inline docs/LabHome/Onboarding/analyst_onboarding.md:160]: but maybe not sitting and reading it cover to cover?

File: docs/LabHome/Onboarding/analyst_onboarding.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+Additionally, check out the [Nature Methods paper](https://www.nature.com/articles/s41592-018-0235-4) written for the software.
+
+#### Diffusion Data
+
+For diffusion imaging, we recommend `QSIPrep`, developed primarily by Dr. Matt Cieslak (manuscript in preparation). Learn about `QSIPrep` at its official [readthedocs site](https://qsiprep.readthedocs.io/en/latest/).
+
+#### ASL Data
+
+For diffusion imaging, we recommend `ASLPrep`<sup>[1](#myfootnote1)</sup>, developed primarily by Dr. Azeez Adebimpe (manuscript in preparation). Learn about `ASLPrep` at its official [readthedocs site](https://aslprep.readthedocs.io/en/latest/).
```
Comment (mattcieslak) [inline docs/LabHome/Onboarding/analyst_onboarding.md:219]: for Arterial spin labeling*

**Review by mattcieslak — CHANGES_REQUESTED** [review]
Looks great! Just a few suggestions

**Comment by TinasheMTapera** [comment]
All helpful comments, I'll make these changes and deploy

**Pull request merged**